src/scheduler/gc_work.rs

```diff
@@ -421,31 +421,40 @@ pub trait ProcessEdgesWork:
         // So maximum 1 `ScanObjects` work can be created from `nodes` buffer
     }
 
-    /// Create a new scan work packet. If SCAN_OBJECTS_IMMEDIATELY, the work packet will be executed immediately, in this method.
+    /// Start the a scan work packet. If SCAN_OBJECTS_IMMEDIATELY, the work packet will be executed immediately, in this method.
```

Comment:
Actually "Start" is not accurate. If not `SCAN_OBJECTS_IMMEDIATELY`, the work packet doesn't start until it is scheduled on a GC worker.

"Create a new" is more accurate in this context, at least it doesn't imply when it is executed.

Reply:
The method below this method is called `create_scan_work()`. We could call this `start_or_dispatch_scan_work()`, or simply `dispatch_scan_work()`.